src/redist/targets/GenerateLayout.targets

```diff
@@ -94,7 +94,9 @@
 
     <PropertyGroup>
       <CoreSetupRootUrl>$(CoreSetupBlobRootUrl)Runtime/</CoreSetupRootUrl>
+      <CoreSetupRootUrl Condition=" '$(DotNetBuildFromSource)' == 'true' AND '$(DotNetBuildOffline)' == 'true' ">$(CoreSetupBlobRootUrl)</CoreSetupRootUrl>
```

Comment:
The `AND '$(DotNetBuildOffline)' == 'true'` can be removed from this file once the two patches included in the PR are migrated back to the repos and picked up by installer.

Reply:
Did we file an issue for this?